modules/cluster-logging-updating-logging-46.adoc

```diff
@@ -0,0 +1,319 @@
+green  open   infra-000001                                                          eH53BQ-bSxSWR5xYZB6lVg   3 1       341800            0        443            220
+green  open   .kibana-6                                                             RVp7TemSSemGJcsSUmuf3A   1 1            4            0          0              0
+green  open   infra-000011                                                          J7XWBauWSTe0jnzX02fU6A   3 1       226100            0        293            146
+green  open   app-000001                                                            axSAFfONQDmKwatkjPXdtw   3 1       103186            0        126             57
+green  open   infra-000016                                                          m9c1iRLtStWSF1GopaRyCg   3 1        13685            0         19              9
+green  open   infra-000002                                                          Hz6WvINtTvKcQzw-ewmbYg   3 1       228994            0        296            148
+green  open   project.qt.2c05acbd-bc12-4275-91ab-84d180b53505.2020.06.30            MUm3eFJjSPKQOJWoHskKqw   3 1        12262            0         14              7
+green  open   infra-000013                                                          KR9mMFUpQl-jraYtanyIGw   3 1       228166            0        298            148
+green  open   audit-000001                                                          eERqLdLmQOiQDFES1LBATQ   3 1            0            0          0              0
+----
+====
+
+.. Verify that the log collector is updated to 4.6:
++
+[source,terminal]
+----
+$ oc get ds fluentd -o json | grep fluentd-init
+----
++
+You should see a `fluentd-init` container.
++
+.Example output
+[source,terminal]
+----
+"containerName": "fluentd-init"
+----
+
+.. Verify that the log visualizer is updated to 4.6 using the Kibana CRD:
++
+[source,terminal]
+----
+$ oc get kibana kibana -o json
+----
++
+You should see a Kibana pod with the `ready` status:
++
+.Example output with a ready Kibana pod
+[%collapsible]
+====
+[source,json]
+----
+[
+{
+"clusterCondition": {
+"kibana-5fdd766ffd-nb2jj": [
+{
+"lastTransitionTime": "2020-06-30T14:11:07Z",
+"reason": "ContainerCreating",
+"status": "True",
+"type": ""
+},
+{
+"lastTransitionTime": "2020-06-30T14:11:07Z",
+"reason": "ContainerCreating",
+"status": "True",
+"type": ""
+}
+]
+},
+"deployment": "kibana",
+"pods": {
+"failed": [],
+"notReady": []
+"ready": []
+},
+"replicaSets": [
+"kibana-5fdd766ffd"
+],
+"replicas": 1
+}
+]
+----
+====
+
+.. Verify the Curator is updated to 4.6:
++
+[source,terminal]
+----
+$ oc get cronjob -o name
+----
++
+.Example output
+[source,terminal]
+----
+cronjob.batch/curator
+cronjob.batch/elasticsearch-im-app
+cronjob.batch/elasticsearch-im-audit
+cronjob.batch/elasticsearch-im-infra
+----
++
+You should see the `elasticsearch-delete-\*` and `elasticsearch-rollover-*` indices approximately 30 minutes after an installation or upgrade.
```

Comment:
What want we to describe here? There isn't any elasticsearch-xxx indices.
Does it means below:
You should see the `elasticsearch-im-app\*` and `elasticsearch-im-audit-*` and `elasticsearch-im-infra' pods approximately 15 minutes after an installation or upgrade.

Reply:
@wgordon17  Can you guide us on Anping's question here? Rick is not available right now.

Reply:
Yeah, the use of `indices` seems odd, considering that you're listing `cronjob`s. On a new installation, I'm not seeing `elasticsearch-` _anything_ in either of the `openshift-logging` or `openshift-operators-redhat` namespace. I'm not sure if that's expected. Either way, I can't validate the correct/expected end state, so I think we'll have to wait for @RiRa12621 because if we publish this and it's wrong, then I 100% would expect customer support cases to come out of this confusion.